### ISSUE TITLE
vcctl failed to load kubeconfig files in `KUBECONFIG`

### ISSUE THREAD

**Issue by hzxuzhonghu** [volcano-sh/volcano#353]
Running the following, the env `KUBECONFIG` does not take effect.

```
$ export KUBECONFIG=/root/.kube/kind-config-volcano
$ _output/bin/linux/amd64/vcctl queue list

Failed to list queue: stat /root/.kube/config: no such file or directory
```

**Comment by TommyLike** [comment]
Kubeconfig has to be specified distinct. Need fix.
```
Flags:
      --all-namespaces      list jobs in all namespaces
  -h, --help                help for list
  -k, --kubeconfig string   (optional) absolute path to the kubeconfig file (default "/root/.kube/config")
  -s, --master string       the address of apiserver
  -n, --namespace string    the namespace of job (default "default")
  -S, --scheduler string    list job with specified scheduler name
      --selector string     fuzzy matching jobName
```

@asifdxtreme

**Comment by hzxuzhonghu** [comment]
i think this is fixed by #478
